include/znc/Chan.h

```diff
@@ -77,7 +77,9 @@ class CChan : private CCoreTranslationMixin {
 
     // Modes
     void SetModes(const CString& s);
+    void SetModes(const VCString& vsModes);
```

Comment:
Signature of this method indicates that every mode is in a separate string. How are parameters passed? e.g. which nick got +o and which mask got +b?

Reply:
This is a vector of `{"+oosvb", "Op1", "Op2", "VoicedUser", "BannedUser!*@*"}`, acting the same as `SetModes(const CString&)` but taking the arguments directly from the message instead of concating them and re-splitting them inside of `SetModes` as is currently done.

Reply:
Could you add a comment explaining that?

Also, since the first element is special, probably it belongs to a separate parameter